tests/plugins_tests/test_scheduler_plugins.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from vllm.core.scheduler import Scheduler
+
+
+class DummyScheduler(Scheduler):
+
+    def schedule(self):
+        raise Exception("Exception raised by DummyScheduler")
+
+
+def test_scheduler_plugins():
+    import pytest
+
+    from vllm.engine.arg_utils import EngineArgs
+    from vllm.engine.llm_engine import LLMEngine
+    from vllm.executor.uniproc_executor import UniProcExecutor
+    from vllm.sampling_params import SamplingParams
+
+    with pytest.raises(Exception) as exception_info:
+
+        engine_args = EngineArgs(
+            model="facebook/opt-125m",
+            enforce_eager=True,  # reduce test time
+        )
+        vllm_config = engine_args.create_engine_config()
+        vllm_config.parallel_config.scheduler_cls = DummyScheduler
```

Comment:
we should have a top-level argument for it, e.g. from `--scheduler_cls "mod.name"` or `EngineArgs(scheduler_cls="mod.name")`